src/main/java/bio/terra/cli/service/GoogleGcsBuckets.java

```diff
@@ -0,0 +1,38 @@
+package bio.terra.cli.service;
+
+import bio.terra.cli.service.utils.CrlUtils;
+import bio.terra.cloudres.google.storage.BlobCow;
+import bio.terra.cloudres.google.storage.BucketCow;
+import bio.terra.cloudres.google.storage.StorageCow;
+import com.google.api.gax.paging.Page;
+import com.google.auth.oauth2.GoogleCredentials;
+import java.util.stream.StreamSupport;
+
+public class GoogleGcsBuckets {
```

Comment:
nit: public class should have a description. Also perhaps rename this class to `GoogleCloudStorage` since that's the service name.